src/App/App.js

```diff
@@ -10,25 +11,41 @@ const beValues = [
   {
     id: '0',
     caption: 'Card one header',
-    content: loremIpsum
+    content: loremIpsum,
+    selected: true
   }, {
     id: '1',
     caption: 'Card two header',
-    content: loremIpsum
+    content: loremIpsum,
+    selected: true
   }, {
     id: '2',
     caption: 'Card three header',
-    content: loremIpsum
+    content: loremIpsum,
+    selected: false
   }
 ];
 
+const beValuesTransformed = {};
+beValues.forEach(one => beValuesTransformed[one.id] = one);
+
 
 function App() {
-  const cards = beValues.map(one =>
+  const [cardsState, setCardsState] = useState(beValuesTransformed);
+
+  const cardsRendered = Object.entries(cardsState).map(([key, one]) =>
```

Comment:
Can you please avoid it. Can you please imagine how it is read by another developer)

I think it'll be enough to keep values in state and for render map only cardsState.